data/marauders.txt

```diff
@@ -403,7 +403,7 @@ ship "Marauder Firebird"
 	explode "medium explosion" 24
 	explode "large explosion" 8
 	"final explode" "final explosion medium"
-	description "By the looks of the modification that have taken place, you suspect that this ship contains no original parts. With extra shield emitters, hull plating, and outfit space, this half-millennium young ship is finding new life with its heavy modification."
+	description "By the looks of the modification that have taken place, you suspect that this ship contains no original parts. With extra shield emitters, hull plating, and outfit space, this half-millennium-old design is finding new life with its heavy modification."
```

Comment:
I'm not a native English speaker. But using the word young here sounds acceptable to me.
(Acceptable in the same way that my grandfather used to claim that he was 80 years young. Everybody understands that he meant old, but it suggests a hidden meaning of enjoying alive and feeling young.)

Reply:
Just because it wasn't ungrammatical doesn't mean it wasn't awkward; there's no need to *imply* that it's enjoying life and feeling young when it's immediately followed by the explicit statement that it's finding new life.

Reply:
Either one is fine. Judgment call, though I slightly prefer the changed version.